README.md

```diff
@@ -55,7 +56,13 @@ Install npm dependencies and start it up:
 - `yarn`
 - `yarn start`
 
-This will start a local development server on port 5000 (http://localhost:5000).
+This will start a local development server on port 5000 (http://localhost:5000). 
+
+**Note**: For fetching from Bugzilla, this application rely on 
```

Comment:
Add a new section `Web Server`before `Environment`. You can have something like:

Codetribute relies on two servers, namely, GitHub’s GraphQL API v4 (managed by GitHub) and [bugzilla-graphql-gateway](http://github.com/mozilla-frontend-infra/bugzilla-graphql-gateway). The latter is required to perform queries to the Bugzilla API. For a local setup, clone the repo and follow the instruction for starting it prior to launching this application. You will need to launch the bugzilla-graphql-gateway in a terminal instance separate from this application in order to run both simultaneously.